src/src/WebServer/HTML_wrappers.cpp

```diff
@@ -286,7 +286,11 @@ void html_add_JQuery_script() {
 
 #if FEATURE_CHART_JS
 void html_add_ChartJS_script() {
-  addHtml(F("<script src=\"https://cdn.jsdelivr.net/npm/chart.js\"></script>"));
+  // To update the CDN link go to: https://www.chartjs.org/docs/latest/getting-started/installation.html
+  // - Select a CDN (jsdelivr is fine)
+  // - Select the chart.js file (may be called chart.umd.min.js) and copy the url
+  // - Replace the url in below script src element, keeping the quotes
+  addHtml(F("<script src=\"https://cdn.jsdelivr.net/npm/chart.js@4.1.2/dist/chart.umd.min.js\"></script>"));
```

Comment:
Perhaps we should have this link as a define which can be overridden by a Custom.h file

Reply:
I'll add that